product_docs/docs/eprs/6.2/02_overview/04_design_replication_system/03_restrictions_on_replicated_database_objects.mdx

```diff
@@ -95,14 +107,22 @@ Postgres tables that include the following data types cannot be replicated to a
 -   `ABSTIME`
 -   `ACLITEM`
 -   `CHKPASS`
+-   Geometric data types (such as `LINE`, `PATH`, `POLYGON`)
 -   `CIRCLE`
 -   `CUBE`
--   `JSON`
+-   `JSON` 
+-   `JSONB`
 -   `ROWID`
+-   `tsvector`, `tsquery`, `txid_snapshot`, `pg_lsn`
 -   `SEG`
 -   `Any ARRAY` data type (that is, defined as `data_type[]`)
+-   `ENUM`, composite type, range data type (such as `int4range`, `tstzrange`, `numrange`, `daterange`), `mcaddress8`
 -   Any user-defined data type (that is, defined as `CREATE TYPE type_name`)
 
+!!! note
+Tables containing columns of datatype `mcaddress8` can be replicated from PostgreSQL to PostgreSQL. However, replication from  PostgreSQL to  EDB Postgres Advanced Server and between EDB Postgres Advanced Servers is not supported.
+!!!
+
```

Comment:
The ending Admonitions (!!!) are not needed I think. Also, the text is to be aligned with "note"